promgen/migrations/0018_sender_enabled.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.10 on 2020-03-14 03:10
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('promgen', '0017_alter_alertlabel_field'),
```

Comment:
Will need to rename this to `0019_sender_enabled.py` and fix this to 

```suggestion
        ('promgen', '0018_default_exporter_scheme'),
```

due to the earlier PR that was already merged 🙇